tests/components/recorder/test_statistics.py

```diff
@@ -3740,3 +3741,22 @@ async def test_get_statistics_service_missing_mandatory_keys(
             return_response=True,
             blocking=True,
         )
+
+
+# The STATISTIC_UNIT_TO_UNIT_CONVERTER keys are sorted to ensure that pytest runs are
+# consistent and avoid `different tests were collected between gw0 and gw1`
+@pytest.mark.parametrize("uom", sorted(STATISTIC_UNIT_TO_UNIT_CONVERTER))
```

Comment:
Damn! we need key added!
And I'm not adding as a suggestion as it would fail ruff

```python
@pytest.mark.parametrize("uom", sorted(STATISTIC_UNIT_TO_UNIT_CONVERTER, key=lambda x: (x is None, x)))
```

Reply:
pushed an update, to fix this